### ISSUE TITLE
(SUGGESTION) Achievements

### ISSUE THREAD

**Issue by R1verwood** [pokeclicker/pokeclicker#1573]
There is a lot of Achievements to be done, especially when you are doing multiple at once!

1) If there was an option to add more Achievements to view, It wouldn't become so tedious, completing quests. 
     
For Example, The dungeons, route kills, and Gyms. it would be nice to have 2-3 Achievements in view at once so once you are done with one, you don't have to constantly click the achievements and look for the one you want to do next. I can have more fun being able to see the bars increasing and either disappear once one is done or have the completed stats on them.

2) A fun and interesting feature for the achievements would be if the "Chain" Quests go to the next achievements after completing the previous one. 
    
For Example,

Explorer
V
Quest Complete
V
Achievements change to Expert
V
Quest Complete
V 
Achievement changes to Hermit
V 
Quest Complete
V
After the "chain", The quest freezes as Complete

**Comment by GarrikFel** [comment]
I'm not a big fan of the second suggestion.  Part of doing achievements is that bit of dopamine you get from achieving it.  If the quests chain into each other, then they switch without you even noticing them.  It would be the same thing as just showing the "hermit" version of the quest the entire time, but with a lot less work for everyone involved.

**Comment by jk13pclick** [comment]
Approving Part 2 of the suggestion, we would like to see the Achievement modal be condensed and Part 2 is a good way of doing so. We do not agree with GarrikFel's assessment. 

Adding more tracked achievements to the view would likely clutter the main UI, but if they are independently collapsible that could work.

Please rename this issue to reflect the provided feedback for a more accurate description of approved ideas. Thank you!
